lib/routes/errors.js

```diff
@@ -21,16 +21,7 @@ module.exports = application => {
      * @param {object} res - HTTP Response
      */
     app.use((req, res) => {
-      const message = _.get(req.session, '404.message', `Not Found: ${req.url}`);
-      const safe = _.get(req.session, '404.safe');
-
-      _.unset(req.session, '404');
-
-      res.status(404);
-      res.render('404', {
-        message,
-        safe,
-      });
+      return routes.notFound(req, res);
```

Comment:
Can we call this `routes.missing` instead of `notFound` just so we can avoid `camelCase`?